src/components/common/LoginPages/layout/splash-container.tsx

```diff
@@ -0,0 +1,54 @@
+import { motion } from "framer-motion";
+import { LanguageDropdown } from "../../localization/language-dropdown";
+import { SplashScreenProps } from "../../../../types/loginTypes";
+
+export const SplashScreen = ({ children, isAnimationEnabled }: SplashScreenProps) => {
+    return (
+        <div className="flex min-h-screen w-screen bg-white">
+            <div className="relative flex w-full items-center justify-center bg-primary-100">
+                <motion.div
+                    initial={
+                        isAnimationEnabled ? { backgroundColor: "#FDEDD7", zIndex: "100" } : {}
+                    }
+                    animate={
+                        isAnimationEnabled
+                            ? { backgroundColor: "rgba(255, 255, 255, 0)", zIndex: "auto" }
+                            : {}
+                    }
+                    transition={{
+                        duration: 1,
+                        delay: 1.25,
+                        ease: "easeInOut",
+                    }}
+                    className="fixed left-0 top-0 h-screen w-screen"
+                >
+                    <motion.img
+                        src="/svgs/login/ssdc_logo.svg"
+                        alt="logo"
+                        initial={
+                            isAnimationEnabled
+                                ? { x: "35vw", y: "25vh", scale: 1 }
+                                : { x: 32, y: 32, scale: 0.25 }
+                        }
+                        animate={
+                            isAnimationEnabled
+                                ? { x: 32, y: 32, scale: 0.25 }
+                                : { x: 32, y: 32, scale: 0.25 }
+                        }
+                        transition={{
+                            duration: 0.75,
+                            delay: 1,
+                            ease: "easeInOut",
+                        }}
+                        className="left-8 top-8 size-full max-h-80 max-w-80 origin-top-left object-cover"
+                    />
+                </motion.div>
+                <img src="/svgs/login/login-image.svg" alt="login image" width={400} height={400} />

```

Comment:
don't use it like this instead add this image in svgs folder and import from ts file the include it like this and don't include width and height.